### ISSUE TITLE
QueryException when accessing hub when using PostgreSQL

### ISSUE THREAD

**Issue by Xenovalis** [getcandy/getcandy#11]
```
Illuminate\Database\QueryException

SQLSTATE[42703]: Undefined column: 7 ERROR: column "count" does not exist LINE 1: ...null group by "store_order_addresses"."id" having "count" <=... ^ (SQL: select count(*) as aggregate from (select COUNT(*) count, "store_order_addresses"."contact_email" from "store_order_addresses" left join store_order_addresses address_join on "address_join"."contact_email" = "store_order_addresses"."contact_email" where "store_order_addresses"."type" = billing and "store_order_addresses"."created_at" between 2021-12-20 00:00:00 and 2021-12-26 00:00:00 and "store_order_addresses"."contact_email" is not null group by "store_order_addresses"."id" having "count" <= 1) as "temp_table")
```
https://github.com/getcandy/getcandy/blob/947c6b3bad9da199dd8da2aa858161af7b80f7e1/src/Http/Livewire/Dashboard.php#L90

This issue only occurs when using PostgreSQL. The error from #9 is given when using MariaDB.

**Comment by Xenovalis** [comment]
Also, `DATE_FORMAT` only really exists in MySQL and is not in PostgreSQL.

**Comment by alecritson** [comment]
There is a PR for this on #12

**Comment by alecritson** [comment]
This should be resolved in [2.0-beta4](https://github.com/getcandy/getcandy/releases/tag/2.0-beta4) If it's still an issue please re open :)
